flathunter/heartbeat.py

```diff
@@ -1,11 +1,15 @@
 """Providing heartbeat messages"""
-from flathunter.logging import logger
+from typing import Union
+
+from flathunter.abstract_notifier import Notifier
 from flathunter.config import Config
+from flathunter.logging import logger
+from flathunter.sender_apprise import SenderApprise
 from flathunter.sender_mattermost import SenderMattermost
 from flathunter.sender_telegram import SenderTelegram
-from flathunter.sender_apprise import SenderApprise
 
-def interval2counter(interval):
+
+def interval2counter(interval: str) -> Union[None, int]:
```

Comment:
Why are you using ```Union``` here?

Reply:
The outcome of this function is either `int` or `None`. see below: https://github.com/flathunters/flathunter/blob/0b5445de666fa660d43def10bf4d8f068145632f/flathunter/heartbeat.py#L10-L11